Makefile

```diff
@@ -20,12 +20,6 @@ ingestor-init:
 	$(docker_compose) --profile ingestor up -d
 	cd ./test_infra/; ./seed_moto.sh
 
-.PHONY: api-init
-api-init:
-	docker compose --profile apiv1 up -d
-	docker compose cp test_infra/sql db:/tmp/sql
-	docker compose exec db sh -c 'cat /tmp/sql/seed_db_data.sql | psql postgres://postgres:postgres@127.0.0.1:5432/cryoet'
```

Comment:
Right now the docker-compose.yml file has the db come up with a `cryoet` schema by default for the v1 api (that we don't need anymore), and then there's a step in this Makefile that creates the `cryoetv2` schema for apiv2. I think we can update the docker-compose.yml file to just start up with a `cryoetv2` database by default and remove the steps that create the second DB from this Makefile